packages/logo/composer.json

```diff
@@ -4,6 +4,7 @@
 	"type": "library",
 	"license": "GPL-2.0-or-later",
 	"require": {},
+	"version": "1.0.0",
```

Comment:
I'm hoping we'll soon land the mirroring solution and release the logo as a proper package, so we won't have to do this.